unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/MatrixInitializationBridge.cs

```diff
@@ -0,0 +1,26 @@
+using DCL.Social.Friends;
+using JetBrains.Annotations;
+using System;
+using UnityEngine;
+
+namespace MainScripts.DCL.Controllers.FriendsController
+{
+    public class MatrixInitializationBridge : MonoBehaviour
+    {
+        public event Action<string> OnReceiveMatrixAccessToken;
+
+        public static MatrixInitializationBridge i { get; private set; }
```

Comment:
i think we can follow a better pattern by doing similar to `WebInterfaceChatBridge`:
```
public static MatrixInitializationBridge GetOrCreate()
        {
            var bridgeObj = SceneReferences.i?.bridgeGameObject;
            return bridgeObj == null
                ? new GameObject("Bridges").AddComponent<MatrixInitializationBridge>()
                : bridgeObj.GetOrCreateComponent<MatrixInitializationBridge>();
        }
```
If you make this change, you have to be careful to change it kernel aswell, send a message to `Bridges` instead of `Main`